setup.py

```diff
@@ -45,8 +45,7 @@ def package_files(package_dir, subdirectory):
     "package_data": {"hera_opm": data_files},
     "install_requires": ["toml>=0.9.4"],
     "zip_safe": False,
-    "setup_requires": ["pytest-runner", "toml>=0.9.4"],
-    "tests_require": ["pytest", "toml>=0.9.4"],
+    "extras_require": {"test": ["pytest", "pytest-cov"]},
```

Comment:
is there a reason this isn't in `setup.cfg`?

Reply:
Not a good one, I will move it over.